databases/cassandra/cassandra_test.go

```diff
@@ -58,6 +58,10 @@ func TestHappyPath(t *testing.T) {
 		t.Errorf("Got unexpected error when initializing cassandra cluster - %s", err)
 	}
 
+	if db == nil {
+		t.Fatalf("Database has not been configured , db = t %v", db)
+	}
+
```

Comment:
Maybe we should make line 58 a `t.Fatalf` too. That would cover this scenario and be a bit less code. But good catch either way.

Reply:
i'll make that change ,

the t.Error used to failed the test case , i guess it is in new release

Reply:
Yeah, `t.Error` will only fail the test case but `t.Fatal` will stop executing the tests. It's cool and easy.

Reply:
i made all the changes ,